engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/content-hosts-bulk-packages-modal.controller.js

```diff
@@ -134,7 +135,9 @@ angular.module('Bastion.content-hosts').controller('ContentHostsBulkPackagesModa
                 $scope.packageActionFormValues.remoteAction = 'package_' + action;
             }
 
-            if (selectedHosts.included.ids) {
+            if ($scope.allHostsSelected) {
+                $scope.packageActionFormValues.scopedSearch = selectedHosts.included.search;
+            } elseif (selectedHosts.included.ids.length > 0) {
```

Comment:
I guess it should be 'else if' :roll_eyes:

Reply:
yeah the linter smacked my hand